tox.ini

```diff
@@ -16,6 +16,13 @@ deps = -rrequirements/requirements.txt
 #commands = pytest --channel="nightly" --pref-set="moztestpub" --driver=Firefox tests/test_filesize.py -s
 commands = pytest --channel="nightly" --pref-set="base" tests/test_tracking_protection.py -s
 
+[testenv:itisatrap]
```

Comment:
this has been modified, but I'm not even invoking tox.  we prob want to consider just removing it.